### ISSUE TITLE
VS crashes if you put multiple frameworks inside <TargetFramework> element

### ISSUE THREAD

**Issue by mavasani** [dotnet/sdk#657]
Ported from https://github.com/dotnet/roslyn-project-system/issues/1175

**Repro Steps**
Create a ClassLibrary (.NetCore)
Edit csproj  
Change 
```
<TargetFramework>netstandard1.4</TargetFramework> to
<TargetFramework>netstandard1.4;net452</TargetFramework>
```

**Actual**
![image](https://cloud.githubusercontent.com/assets/12449387/21993741/468c7bd0-dbd1-11e6-9a5a-708cc10ad4e7.png)

**Expected**
The warning here is expected and you should see that warning once.
The right edit is `<TargetFrameworks>netstandard1.4;net452</TargetFrameworks>`

Note: If it doesn't crash right away, try closing the csproj and try opening for edit again

BuildInfo : 26114.2.d15rel

**Comment by mavasani** [comment]
@nguerrera mentioned we should add a check in targets to identify this and give a better error message indicating that the user probably intended to use TargetFrameworks, not TargetFramework

**Comment by srivatsn** [comment]
Dupe of https://github.com/dotnet/sdk/issues/385

**Comment by mavasani** [comment]
The crash seems to be due to us appending the TargetFramework to the OutputPath even when TF inference fails: https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.TargetFrameworkInference.targets#L93

Lets use this bug to track the VS crash and #385 to track the error message improvement.

**Comment by srivatsn** [comment]
Closed by #660
